themes/bootstrap3/templates/Recommend/TopFacets.phtml

```diff
@@ -49,4 +49,4 @@
       <? endif; ?>
     </div>
   <? endforeach; ?>
-<? endif; ?>
+<? endif; ?>
```

Comment:
I think this should be reverted -- it's removing one of the line endings you fixed a week or two ago.